Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml

```
@@ -0,0 +1,26 @@
+- type: entity
+  name: Centcomm jello
+  parent: FoodsnackBase
+  id: CentcommJello
+  description: Now 30 percent more air for 120 percent of the price
+  components:
+    - type: FlavorProfile
+      flavors:
+        - overpriced
+        - Jello
+    - type: Sprite
+      state: jello
+    - type: SolutionContainerManager
+      solutions:
+        food:
+          maxVol: 1
```

```suggestion
          maxVol: 3
```
The reagents that compose the jello exceed the capacity of the jello.